python/examples/use_cases/github_issue_manager.py

```diff
@@ -0,0 +1,109 @@
+import asyncio
+import os
+import sys
+import traceback
+
+from dotenv import load_dotenv
+from mcp import ClientSession, StdioServerParameters
+from mcp.client.stdio import stdio_client
+
+from beeai_framework.agents.react.agent import ReActAgent
+from beeai_framework.agents.types import AgentExecutionConfig
+from beeai_framework.backend.chat import ChatModel
+from beeai_framework.backend.message import SystemMessage, UserMessage
+from beeai_framework.emitter.emitter import EventMeta
+from beeai_framework.errors import FrameworkError
+from beeai_framework.memory.token_memory import TokenMemory
+from beeai_framework.tools.mcp_tools import MCPTool
```

Comment:
```suggestion
from beeai_framework.tools.mcp import MCPTool
```
to handle refactor in main